orders/test/interface/conftest.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from collections import namedtuple
+
+from nameko.standalone.rpc import ServiceRpcProxy
+from nameko.testing.services import replace_dependencies
+
+from orders.service import OrdersService
+
+
+@pytest.fixture
+def config(rabbit_config, db_url):
+    orders_config = rabbit_config.copy()
+    orders_config['DB_URIS'] = {'orders:Base': db_url}
+    return orders_config
+
+
+@pytest.fixture
+def container(container_factory, config):
+    ServiceMeta = namedtuple(
```

Comment:
There's a newer version of this pattern that allows replacement values to be specified.